src/cgame/cg_buildable.cpp

```diff
@@ -2088,6 +2094,7 @@ static int CG_SortDistance( const void *a, const void *b )
 CG_PlayerIsBuilder
 ==================
 */
+
```

Comment:
Need to fix regex? We shouldn't be adding extra lines everywhere

Reply:
that was just making comments consistent: in this kind of file there is random blank lines (or not) after each comment, it was half multiline comment plus space and half multiline comment without space… not very costly to unify that (i.e. I wrote the regexp to do that).

Reply:
We should do the opposite...remove the extra newline between comments and the start of the function.

Reply:
well, it's not related to licensing so it can be submitted as a separate PR, I removed that commit from this PR history.